test/E2ETest/Microsoft.Test.E2E.AspNet.OData/ComplexTypeInheritance/ComplexTypeInheritanceTests.cs

```diff
@@ -539,6 +649,63 @@ public async Task PutCollectionComplexTypeProperty(string modelMode)
                 contentOfString));
         }
 
+        [Theory]
+        [MemberData(nameof(NewCollectionMembers))]
+        // POST ~/Windows(3)/OptionalShapes
+        public async Task PostToCollectionComplexTypeProperty(string modelMode, string jObject, string targetPropertyResource, bool returnRepresentation)
+        {
+            //Arrange
+            string serviceRootUri = string.Format("{0}/{1}", BaseAddress, modelMode).ToLower();
+            string requestUri = serviceRootUri + "/Windows(3)/"+ targetPropertyResource;
+            int count = 0;
+
+            //Setup the post request
+            var requestForPost = new HttpRequestMessage(HttpMethod.Post, requestUri);
+            requestForPost.Content = new StringContent(content:jObject, encoding: Encoding.UTF8, mediaType: "application/json");
+            if (returnRepresentation)
+            {
+                requestForPost.Headers.Add("Prefer", "return=representation");
+            }
```

Comment:
Move these codes after the "Get request"

Reply:
Done